src/main/java/org/jabref/gui/texparser/ParseTexResultViewModel.java

```diff
@@ -39,4 +64,17 @@ public void activeReferenceChanged(ReferenceViewModel reference) {
             citationList.setAll(reference.getCitationList());
         }
     }
+
+    /**
+     * Search and import unknown references from associated BIB files.
+     */
+    public void importButtonClicked() {
+        ImportEntriesDialog dialog = new ImportEntriesDialog(databaseContext, BackgroundTask.wrap(() ->
+                new ArrayList<>(resolverResult.getNewEntries())));
+
+        dialog.setTitle(Localization.lang("Import entries from LaTeX files"));
+        dialog.showAndWait();
+
+        close.set(true);
```

Comment:
I don't understand this close property.

Reply:
D'oh. It is legacy code from an intermediate implementation. I will remove it.